packages/effect/src/internal/scopedRef.ts

```diff
@@ -41,11 +41,12 @@ export const fromAcquire = <A, E, R>(
         core.flatMap((value) =>
           circular.makeSynchronized([newScope, value] as const).pipe(
             core.flatMap((ref) => {
-              const scopedRef: ScopedRef.ScopedRef<A> = {
-                [ScopedRefTypeId]: scopedRefVariance,
-                pipe() {
-                  return pipeArguments(this, arguments)
+              const scopedRef = {
```

Comment:
Same as resource, move the prototype out so it is shared by all instances.

Reply:
done